project/plugins.sbt

```diff
@@ -7,9 +7,8 @@ libraryDependencies ++= Seq(
   "org.joda" % "joda-convert" % "1.7",
 )
 
-resolvers ++= Seq(
+resolvers ++= Resolver.sonatypeOssRepos("releases") ++ Seq(
   Classpaths.typesafeReleases,
-  Resolver.sonatypeRepo("releases"),
```

Comment:
In [February 2021](https://central.sonatype.org/news/20210223_new-users-on-s01/) Sonatype started spreading its OSS Maven artifact hosting across 2 repository hosts, so we need to use the plural `sonatypeOssRepos` (introduced with https://github.com/sbt/librarymanagement/pull/393) rather than the deprecated singular `sonatypeRepo`.